src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Create/ConditionalCreateResourceHandler.cs

```diff
@@ -44,8 +43,9 @@ public async Task<UpsertResourceResponse> Handle(ConditionalCreateResourceReques
             EnsureArg.IsNotNull(message, nameof(message));
 
             SearchResult results = await _searchService.SearchAsync(message.Resource.InstanceType, message.ConditionalParameters, cancellationToken);
+            SearchResultEntry[] matchedResults = results.Results.Where(x => x.SearchEntryMode == ValueSets.SearchEntryMode.Match).ToArray();
```

Comment:
If I search with `_summary=count` would this force execution?

Reply:
I don't think either of these options are covered by the spec, but possibly we should filter out these parameters

Reply:
Yea, my question is more around if we would execute the search with those parameters thus possibly having an unexpected outcome.